### DISCUSSION TITLE
Issue in creating map

### DISCUSSION THREAD

**Discussion by saxenasatyam** [pik-piam/magclass#90] [General]
Hi,

So, after using plotmap2, I am able to create a map but the output is coming on the world map. I want to make it for IND and y1995 to y2050 only. 
Filter and subset function is not working in this.  
How to make a country-based and year based map using plotmap2?

**Comment by tscheypidi** [comment]
not sure, if that is supported. I would recommend to to convert your magpie object instead to raster and use the raster plotting facilities which are more optimized for map data.

```
library(magclass)
x <- magpiesets::addLocation(x)
r <- as.RasterBrick(x)
```
